spec/features/post_feature_spec.rb

```diff
@@ -0,0 +1,49 @@
+# Flows
+# 1. Create a post
+# Can only create post on own wall
+
+# 2. Delete a post
+# Can only delete own post
+
+require 'rails_helper'
+
+describe Post do
+
+	let(:user){create(:user, :with_profile)}
+	
+	before do
+		visit root_path
+		fill_in 'email', with:  user.email
+		fill_in 'password_digest', with:  user.password
+		click_button 'Log In'
+	end
+
+	context 'creation' do
+		scenario 'can create post on own timeline' do
+			visit user_timeline_path(user)
```

Comment:
this is another place where having a macro would be very useful rather than having these 3 lines be duplicated in multiple places.